package/MDAnalysis/analysis/utils/autocorrelation.py

```diff
@@ -21,44 +21,120 @@
 # J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
 #
 
+"""Autocorrelation utilities --- :mod:`MDAnalysis.analysis.utils.autocorrelation`
+=================================================================================
+
+
+:Authors: Paul Smith & Mateusz Bieniek
+:Year: 2020
+:Copyright: GNU Public License v2
+
+.. versionadded:: 0.21.0
+
+This module is primarily for internal use by other analysis modules. It
+provides functionality for calculating the time autocorrelation function
+of a binary variable (i.e one that is either true or false at each
+frame for a given atom/molecule/set of molecules). This module includes
+functions for calculating both the time continuous autocorrelation and
+the intermittent autocorrelation. The function :func:`autocorrelation`
+calculates the continuous autocorrelation only. The data may be
+pre-processed using the function :func:`intermittency` in order to
+acount for intermittency before passing the results to
+:func:`autocorrelation`.
+
+See Gowers and Carbonne, 2015, (DOI:10.1063.1.4922445) for a further
```

Comment:
Please cite using reST syntax and put the paper in references.rst (see [References](https://www.mdanalysis.org/mdanalysis/documentation_pages/references.html).

If the whole module is based on the paper, please also add a  *duecredit* citation to the module or the appropriate function (look elsewhere in the code base where `duecredit` is used – you will only need the DOI for that).